src/docs/getting-started/quick-start.md

```diff
@@ -1,72 +1,92 @@
 # Quick start
 
+This guide covers how you can quickly get started using Sealer.
+
+## PREREQUISITES
+1. Installing the sealer locally.
+2. Prepare six hosts that meet the following criteria, and record their IPs and passwo
```

Comment:
this is not prerequisite

Reply:
What does that mean?

Reply:
do we need six hosts  for the prerequisite?